.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Test briefcase-template
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```

Comment:
As noted in the comments in tox.ini, we've found it's safest to run these sorts of tests on the "oldest supported Python" to make sure newer language idiom doesn't leak in.